controlpanel/settings/common.py

```diff
@@ -0,0 +1,400 @@
+import os
+from os.path import abspath, dirname, join
+import sys
+
+from controlpanel.frontend.jinja2 import environment
+from controlpanel.utils import is_truthy
+
+
+# -- Feature flags
+ENABLED = {
+
+    # Enable Kubernetes Role-based Access Control
+    "k8s_rbac": is_truthy(os.environ.get("ENABLE_K8S_RBAC", False)),
+
+    # Enable writes to Kubernetes cluster
+    "write_to_cluster": is_truthy(os.environ.get("ENABLE_WRITE_TO_CLUSTER", True)),
+}
+
+# Name of the deployment environment (dev/alpha)
+ENV = os.environ.get("ENV", "dev")
+
+
+# -- Paths
+
+# Name of the project
+PROJECT_NAME = "controlpanel"
+
+# Absolute path of project Django directory
+DJANGO_ROOT = dirname(dirname(abspath(__file__)))
+
+# Absolute path of project directory
+PROJECT_ROOT = dirname(DJANGO_ROOT)
+
+# Directory to collect static files into
+STATIC_ROOT = join(PROJECT_ROOT, "run", "static")
+
+# Directory for user uploaded files
+MEDIA_ROOT = join(PROJECT_ROOT, "run", "media")
+
+# Django looks in these locations for additional static assets to collect
+STATICFILES_DIRS = [
+    join(PROJECT_ROOT, "static"),
+]
+
+# Django looks in these locations for additional templates
+PROJECT_TEMPLATES = [
+
+    # find local component templates
+    join(DJANGO_ROOT, "frontend", "static", "components"),
+]
+
+
+# -- Application
+
+INSTALLED_APPS = [
+    # Django channels for asynchronous support
+    "channels",
+    # Django eventstream for SSE support
+    "django_eventstream",
+    # Django auth system
+    "django.contrib.auth",
+    # OIDC client
+    "mozilla_django_oidc",
```

Comment:
### 3. Replace custom OIDC client code with `mozilla-django-oidc`
>Lines 62 - 63

I scrapped our custom authorization code in favour of Mozilla's Django OIDC module. There's this Django app and a middleware class.

[Next →](https://github.com/ministryofjustice/analytics-platform-control-panel/pull/610/files#diff-33610d8b51443bfb29fe9662b597d96eR128)